src/batch-client/types.ts

```diff
@@ -21,6 +21,12 @@ export interface AutoCompleteOptions {
 	needExp?: boolean;
 	type?: GalSearchType;
 }
+export interface AutoCompleteGALOptions {
+	limit: number;
+	name: string;
+	needExp: number;
+	type: string;
```

Comment:
this should be an enum, with below possible values
`all|account|resource|group`

Reply:
Changed to enum